N/NOMAD/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "NOMAD"
+version = v"1.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    "https://www.gerad.ca/nomad/Downloads/unix_linux/NOMAD.zip" =>
+    "2134e64d5c8728054a797d0067eb49083d66ca64ed8ad7e75c34700b528aeff3",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+export NOMAD_HOME=${WORKSPACE}/srcdir/nomad.3.9.1
+export PATH=${NOMAD_HOME}/bin:$PATH
+cd $NOMAD_HOME
+./configure
+make
+rm -rf doc
+rm -rf bin
+rm -rf examples
+rm -rf lib
+rm -rf utils
+rm -rf tools
+rm -rf ext/sgtelib/bin
+rm -rf ext/sgtelib/example
+rm -rf ext/sgtelib/matlab_server
+rm -rf ext/sgtelib/user_guide
+cd builds/release/lib
+rm libsgtelib.so
+ln -s ../../../ext/sgtelib/lib/libsgtelib.so libsgtelib.so
+cp -rf ${NOMAD_HOME} ${prefix}
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    Linux(:powerpc64le, libc=:glibc),
+    Linux(:i686, libc=:glibc)
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libnomad", :libnomad),
+    LibraryProduct("libsgtelib", :libsgtelib),
+    ExecutableProduct("nomad", :nomad; dir_path="nomad.3.9.1/builds/release/bin")
```

Comment:
```suggestion
    ExecutableProduct("nomad", :nomad, "nomad.3.9.1/builds/release/bin")
```

Reply:
This was a bug in BB, we're fixing it in https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/664